src/main/java/org/intellij/sdk/codesync/alerts/ActivityAlerts.java

```diff
@@ -102,7 +102,7 @@ public static void showActivityAlert(Project project) {
             return;
         }
 
-        if (!hasActivityInTheLastDay(jsonResponse)) {
+        if (hasActivityInTheLastDay(jsonResponse)) {
```

Comment:
Kindly aslo use following URL for getting team activity
```
/v1/team_activity?tz=Asia/Karachi&source=intellij
```

Reply:
Done